website/docs/cli/configuration/configuration.mdx

```diff
@@ -28,11 +28,30 @@ the base path location of the components, and so forth.
 ## Configuration File (`atmos.yaml`)
 
 The CLI config is loaded from the following locations (from lowest to highest priority):
-
+- Environment variable `ATMOS_CLI_CONFIG_PATH` (the ENV var should point to a folder without specifying the file name . if is set the CLI will only look for the config file in the specified folder)
 - System directory (`/usr/local/etc/atmos/atmos.yaml` on Linux, `%LOCALAPPDATA%/atmos/atmos.yaml` on Windows)
+- `$XDG_CONFIG_HOME/atmos/atmos.yaml` (if `$XDG_CONFIG_HOME` is set)
 - Home directory (`~/.atmos/atmos.yaml`)
 - Current directory (`./atmos.yaml`)
-- Environment variable `ATMOS_CLI_CONFIG_PATH` (the ENV var should point to a folder without specifying the file name)
+- `atmos.d/` directory form base path (if `import` section is not set in the config) For an example, see [`examples/demo-atmos.d`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos.d)
+Additionally, Atmos supports `imports` of other CLI configurations. Use imports to break large Atmos CLI configurations into smaller ones, such as organized by top-level section. Imports are relative to the base path (if `import`  section is set in the config). For an example, see [`examples/demo-atmos-cli-imports`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos-cli-imports)
```

Comment:
```suggestion
Additionally, Atmos supports `imports` of other CLI configurations. Use imports to break large Atmos CLI configurations into smaller ones, such as organized by top-level section. File imports are relative to the base path (if `import`  section is set in the config). All imports are processed and deep-merged in order, so that the last file in the list supersedes settings in the preceding imports. For an example, see [`examples/demo-atmos-cli-imports`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos-cli-imports).
```